app/definitions/rest/v1/omnichannel.ts

```diff
@@ -20,11 +20,7 @@ export type OmnichannelEndpoints = {
 	};
 	'livechat/visitors.info': {
 		GET: (params: { visitorId: string }) => {
-			visitor: {
-				visitorEmails: Array<{
-					address: string;
-				}>;
-			};
+			visitor: ILivechatVisitor;
```

Comment:
We need to evaluate this later. Because the only param `visitorEmails` returned is not the truth!

Reply:
It's probably going to be fixed on our requests effort.